client/src/components/profile-page/Profile.css

```diff
@@ -103,10 +104,10 @@
   word-wrap: break-word;
 }
 .profile-bio {
-  height: 15rem;
+  height: 13rem;
```

Comment:
Crazy good stuff going on here
13's an unlucky number though, so are you sure you don't want to keep it at 15?